### ISSUE TITLE
Unconditional modification of CMAKE_MSVC_RUNTIME_LIBRARY

### ISSUE THREAD

**Issue by ch45er** [NVIDIAGameWorks/RayTracingDenoiser#72]
This one is related to #33.

The `CMAKE_MSVC_RUNTIME_LIBRARY` variable is set unconditionally in the CMake lists, which effectively prohibits passing it as a command line argument (since it gets overwritten in the script):
https://github.com/NVIDIAGameWorks/RayTracingDenoiser/blob/067755e96c70d4122342c61a37124111715c4e57/CMakeLists.txt#L64-L66

It would be better to provide a CMake `option` for this or additionally check if  `CMAKE_MSVC_RUNTIME_LIBRARY` is unspecified at that point.

**Comment by dzhdanNV** [comment]
Would it be enough to use this?
```
if (MSVC AND NOT DEFINED CMAKE_MSVC_RUNTIME_LIBRARY)
    set (CMAKE_MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>")
endif ()
```

**Comment by ch45er** [comment]
Yes, that works for me.

**Comment by dzhdanNV** [comment]
Will be included in the next update.

**Comment by dzhdanNV** [comment]
Fixed in v4.4.0.
